<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1080</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>13</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QGraphicsView" name="graphicsViewPhotoResult">
        <property name="maximumSize">
         <size>
          <width>750</width>
          <height>660</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QGraphicsView {
	background-color: white;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_options" native="true">
        <property name="maximumSize">
         <size>
          <width>270</width>
          <height>660</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
	background-color: #5E5E5E;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_options">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>Настройки загрузки снимка</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="leftMargin">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item>
            <widget class="QLabel" name="label_path">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Путь к изображению:</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLineEdit" name="lineEdit_path">
               <property name="minimumSize">
                <size>
                 <width>195</width>
                 <height>25</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>195</width>
                 <height>25</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_file_dialog">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_stick1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
           <property name="spacing">
            <number>7</number>
           </property>
           <property name="sizeConstraint">
            <enum>QLayout::SetMinimumSize</enum>
           </property>
           <property name="leftMargin">
            <number>7</number>
           </property>
           <property name="topMargin">
            <number>7</number>
           </property>
           <property name="rightMargin">
            <number>7</number>
           </property>
           <property name="bottomMargin">
            <number>7</number>
           </property>
           <item>
            <widget class="QLabel" name="label_choice_model">
             <property name="minimumSize">
              <size>
               <width>220</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>240</width>
               <height>25</height>
              </size>
             </property>
             <property name="text">
              <string>Модель для сегментации:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox_choice_model">
             <property name="minimumSize">
              <size>
               <width>230</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>230</width>
               <height>25</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_stick2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_post_processing">
           <property name="title">
            <string>Постобратка изображения</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_morphologic_operations">
                <property name="maximumSize">
                 <size>
                  <width>220</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Морфологические операции:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_morphologic_operations">
                <property name="minimumSize">
                 <size>
                  <width>215</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>215</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_pick_on_picture">
                <property name="maximumSize">
                 <size>
                  <width>220</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Оставить на изображении:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_pick_on_picture">
                <property name="minimumSize">
                 <size>
                  <width>215</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>215</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QLabel" name="label_transparency">
                  <property name="maximumSize">
                   <size>
                    <width>225</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Прозрачность полученной маски:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>5</number>
                  </property>
                  <property name="rightMargin">
                   <number>5</number>
                  </property>
                  <property name="bottomMargin">
                   <number>5</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_0_procent">
                    <property name="minimumSize">
                     <size>
                      <width>30</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>30</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>0%</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSlider" name="horizontalSlider_transparency">
                    <property name="minimumSize">
                     <size>
                      <width>120</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>100</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_100_procents">
                    <property name="minimumSize">
                     <size>
                      <width>35</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>35</width>
                      <height>25</height>
                     </size>
                    </property>
                    <property name="text">
                     <string>100%</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_delete_objects">
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>30</height>
               </size>
              </property>
              <property name="text">
               <string>Удаление слишком
больших/маленьких объектов</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_border_visualisation">
              <property name="maximumSize">
               <size>
                <width>220</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Визуализация границ</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_stick3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QPushButton" name="pushButton_start">
             <property name="text">
              <string>Начать обработку</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_save">
             <property name="text">
              <string>Сохранить результат</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_reset">
             <property name="text">
              <string>Сбросить настройки</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_main">
    <property name="title">
     <string>Вид</string>
    </property>
    <widget class="QMenu" name="menu_mode">
     <property name="title">
      <string>Режим</string>
     </property>
     <addaction name="action_mode_mask"/>
     <addaction name="action_mode_original"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="action_view_increase"/>
    <addaction name="action_view_decrease"/>
    <addaction name="menu_mode"/>
    <addaction name="action_view_center"/>
    <addaction name="action_view_detail"/>
    <addaction name="action_view_saveView"/>
    <addaction name="action_view_save"/>
   </widget>
   <addaction name="menu_main"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_view_increase">
   <property name="text">
    <string>Увеличить (Ctrl + +)</string>
   </property>
  </action>
  <action name="action_view_decrease">
   <property name="text">
    <string>Уменьшить (Ctrl + -)</string>
   </property>
  </action>
  <action name="action_mode_mask">
   <property name="text">
    <string>Маска (Ctrl + m)</string>
   </property>
  </action>
  <action name="action_mode_original">
   <property name="text">
    <string>Оригинал (Ctrl + o)</string>
   </property>
  </action>
  <action name="action_view_center">
   <property name="text">
    <string>Центрирование (Ctrl + R)</string>
   </property>
  </action>
  <action name="action_view_detail">
   <property name="text">
    <string>Детально (Ctrl + I)</string>
   </property>
  </action>
  <action name="action_view_saveView">
   <property name="text">
    <string>Сохранить вид (Ctrl + F)</string>
   </property>
  </action>
  <action name="action_view_save">
   <property name="text">
    <string>Сохранить результат (Ctrl + S)</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
